docs/decisions/0007-maplibre.md

```diff
@@ -0,0 +1,67 @@
+# Use MapLibre for Map Visualization
+
+## Context and Problem Statement
+
+Styling a map with a large number of features, in a performant way, is difficult. With 200,000+ block groups, our project needs this performance for a smooth user experience. Our current mapping approach, using OpenLayers, is insufficient for this task. We propose switching to MapLibre as a free and open source way to improve the performance and user experience of our map.
+
+## Decision Drivers
+
+- Free and Open Source
+- Good performance in rendering a large number of features
+- Good performance in zooming and panning
+- Easy map styling
+
+## Considered Options
+
+- MapLibre GL JS 1.15.0
+- Mapbox GL JS 2.3.1
+- Mapbox GL JS 1.13.0
+- OpenLayers 6.4.3
+
+## Decision Outcome
+
+Chosen option: [MapLibre 1.15.0](https://github.com/maplibre/maplibre-gl-js), a free and open-source alternative to Mapbox with good performance and standard map styling cabilities, and continued development lifecycle.
+
+### Positive Consequences
+
+- MapLibre is a free and open source alternative
+- Performance is preferable to OpenLayers, as seen in this video: [MapLibre Performance](https://user-images.githubusercontent.com/72811320/124788369-446a6f80-df17-11eb-9dec-6635f8bfc3cb.mp4)
```

Comment:
We should note that this url may change (or go away). Might be better to upload and forget about it on youtube?

Reply:
ah true, I had added the video to the repo but hadn't updated the link